faithcomesbyhearing/dbp
# FCBHDBP-604 FCBHDBP-605 fixes the pagination issue in the playlist endpoint and translate endpoint

# Description
FCBHDBP-604 fixes the pagination issue in the playlist endpoint. FCBHDBP-605 addresses the incorrect values for the 'verse_start' and 'verses' parameters in the translate endpoint.

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-604
https://fullstacklabs.atlassian.net/browse/FCBHDBP-605

## How Do I QA This
- You should run the following postman test and it has to pass (playlist pagination)
https://digitalbibleplatform.postman.co/workspace/DBP4-workspace~0a283f7f-06d9-467d-97f0-fef92d18bd3d/request/12519377-7dd338dd-b3f3-47a0-a023-2b4663037496?active-environment=810fd94b-9392-43e2-9f13-943e8d323135

- You should run the following postman test and it has to pass (Fix values for verse_start and verses parameter in the translate endpoint)
https://digitalbibleplatform.postman.co/workspace/DBP4-workspace~0a283f7f-06d9-467d-97f0-fef92d18bd3d/request/12519377-5efdb481-2849-4dc9-bc44-0c44fd5f56d6?active-environment=652610c6-424b-4d3f-9dd9-e4dadc2ca43f
